pkg/scheduler/framework/plugins/nodevolumelimits/csi_test.go

```diff
@@ -792,6 +792,94 @@ func TestCSILimitsAddedPVCQHint(t *testing.T) {
 	}
 }
 
+func TestCSILimitsDeletedVolumeAttachmentQHint(t *testing.T) {
+	tests := []struct {
+		test        string
+		newPod      *v1.Pod
+		existingPvc *v1.PersistentVolumeClaim
+		deletedVA   *storagev1.VolumeAttachment
+		wantQHint   framework.QueueingHint
+	}{
+		{
+			test: "a pod has the same PVC as an existing PVC when VolumeAttachment is deleting",
```

Comment:
Do we need a test case that a pod that has multiple volumes and only 1 of them is PVC